docker-compose.yml

```diff
@@ -152,11 +152,11 @@ services:
       - 3054
     networks:
       - chs-dev
-  api.ch.gov.uk:
+  api-ch-gov-uk:
     image: 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/api.ch.gov.uk:latest
-    container_name: api.ch.gov.uk
+    container_name: api-ch-gov-uk
     labels:
-      - traefik.http.routers.api.ch.gov.uk.rule=Host(`api.ch.gov.uk`)
+      - traefik.http.routers.api-ch-gov-uk.rule=Host(`api.chs-dev`)
```

Comment:
should it not be _api-ch-gov-uk_?

Reply:
For the host? Nope, if you look above, you will see that `ch-gov-uk` is accessible via `chs-dev`, so this follows that pattern

Reply:
Cool